docs/references/express/migrate-from-node-to-express.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Clerk Express SDK
+description: Learn how to migrate from Clerk Node SDK to Express SDK.
+---
+
+# Clerk Express SDK
+
+## Migrating from Clerk Node SDK to Clerk Express SDK
+
+<Steps>
+
+### Uninstall @clerk/clerk-sdk-node and install @clerk/express
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm uninstall @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+yarn remove @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+pnpm remove @clerk/clerk-sdk-node
+```
+
+</CodeBlockTabs>
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+### Require Authentication
+
+**Using Node SDK**
+
+To enforce strict authentication, use `ClerkExpressRequireAuth()`. This middleware will raise an error when an unauthenticated request is made.
```

Comment:
🔧 the `@clerk/clerk-sdk-node` also exports a `requireAuth`. Let's add it here (as we did with `withAuth` in the next section)